docs/dev/build.rst

```diff
@@ -32,3 +32,50 @@ GLUON_BASE_FEEDS
     verbatim. By default, this contains a reference to the Gluon base packages;
     when using the Gluon build system to build a non-Gluon system, the variable
     can be set to the empty string.
+
+Helper scripts
+--------------
+
+Several tasks of the buildprocess have been separated from the Makefile into 
+external scripts, which are stored in the *scripts* directory. This was done to
+ease maintenance of these scripts and the Makefile, by avoiding a lot of escaping.
+These scripts are either bash or lua scripts that run on the build system.
+
+default_feeds.sh
+    Defines the CONSTANT *DEFAULT_FEEDS* with the names of all feeds listed in 
+    *openwrt/feeds.conf.default*
+
+feeds.sh
+    Creates the *openwrt/feeds.conf* file from *FEEDS* and *DEFAULT_FEEDS*. The
+    feeds from *FEEDS* are linked to the matching subfolder of *packages/* and not
+    explicitly defined feeds of *DEFAULT_FEEDS* are setup as dummy (src-dummy).
+    This *openwrt/feeds.conf* is used to reinstall all packages of all feeds with
+    the *openwrt/scripts/feeds* tool.
+
+modules.sh
+    Defines the CONSTANT *GLUON_MODULES* and *FEEDS* by reading the *modules*
+    file of the root-folder and the site-folder. The returned variables look
+    like:
+
+    - *FEEDS*: "*feedA feedB ...*"
+    - *GLUON_MODULES*: "*openwrt packages/feedA packages/feedB ...*"
+
+patch.sh
+    (Re-)applies the patches from the *patches* directory to all *GLUON_MODULES*
+    and checks out the files to the filesystem.
+    This is done for each repo by:
```

Comment:
This sentence doesn't match the following items. Either change the following items to start with "creating", "applying", ..., or change the previous one, for example "For each repo, the following steps are run:".

Reply:
used the "*ing" form ...